linera-base/Cargo.toml

```diff
@@ -44,6 +44,7 @@ getrandom = { workspace = true, optional = true }
 hex.workspace = true
 is-terminal.workspace = true
 linera-witty = { workspace = true, features = ["macros"] }
+once_cell.workspace = true
```

Comment:
[`OnceCell` is in the standard library now.](https://doc.rust-lang.org/std/cell/struct.OnceCell.html)

Reply:
That one isn't safe to use between threads and `static`s require that 🤔

Reply:
Ah, I forgot about this one (and I was reading about it just a week ago).

Reply:
Done in [Review part1](https://github.com/linera-io/linera-protocol/pull/3345/commits/f8ad395f83f175631105fba95a5a046aeb61fd41).